editor/printers/ender3-pro.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
moved from the parent folder. I don't know if that folder is accessible from editor. Let me know if you'd like it to be back in the main folder.